smac/intensification/intensification.py

```diff
@@ -410,15 +412,23 @@ def _race_challenger(self,
                 incumbent=incumbent, challenger=challenger,
                 run_history=run_history,
                 log_traj=log_traj)
+
+            # update intensification stage
             if new_incumbent == incumbent:
                 # move on to the next iteration
                 self.stage = IntensifierStage.RUN_INCUMBENT
                 self.continue_challenger = False
 
             elif new_incumbent == challenger:
+                # New incumbent found
                 incumbent = challenger
-                # New incumbent found. Go to the next stage
-                self.stage = IntensifierStage.RUN_DEFAULT
+                # compare against default configuration if provided, else go to next iteration
```

Comment:
Calling this the default configuration might be confusing. Could you rename this to "basis". Please also change this everywhere if you agree